frontend/src/pages/answer/resultsTable/ResultExplorer.jsx

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable indent, no-use-before-define, func-names, no-return-assign */
```

Comment:
Part of the idea here was so that the user could see which nodes changed between different results. For a result with two nodes, it's harder to see, but for a longer query graph and two results that just have one node different, it'd make more of an impact.

Reply:
Fixed in 3531c7f8aa34bc2d48844ea6ba4988bdd4b574c2